docker/postgres-indexer/README.md

```diff
@@ -0,0 +1,24 @@
+# Postgres Indexing Database Docker
+
+This docker container is for using the postgres indexer.  
+
+To configure postgres indexing change indexer configuration in the `config.toml`
```

Comment:
Please direct them to put this in `custom.toml` so that it won't get stomped out by `provenanced config` stuff.
```suggestion
To configure postgres indexing change indexer configuration in the `custom.toml`
```

Reply:
I was wrong here. This section and fields are part of the `config.toml` template. So they don't need to go in the `custom.toml` file.